DUI3-DX/Connectors/Revit/Speckle.Connectors.RevitShared/Operations/Send/RootObjectBuilder.cs

```diff
@@ -47,7 +49,15 @@ RevitConversionContextStack contextStack
         continue;
       }
 
-      commitObject[obj.UniqueId] = _converter.Convert(obj);
+      // POC: temporary, the converter needs to be
+      try
+      {
+        commitObject[obj.UniqueId] = _converter.Convert(obj);
+      }
+      catch (Exception ex) when (!ex.IsFatal())
+      {
+        Debug.WriteLine($"Exception during conversion: {ex.Message}");
+      }
     }
```

Comment:
Incomplete comment 👍🏻 but the point is clear.

I'm still rather fond of the aggregate exception thing (mostly for debug purposes) as 1 breakpoint at the end of the loop let's you inspect all errors.

Reply:
yeaj I just needed it to not explode as it did as soon as I sent the world and dog